package.json

```diff
@@ -20,14 +21,15 @@
   "devDependencies": {
     "@changesets/cli": "^2.27.10",
     "@eslint/js": "^9.22.0",
+    "concurrently": "^8.0.0",
```

Comment:
Moved concurrently to the root so we can use it without adding it to every package. Note, this should only be done for reused dev dependencies.